src/dbcp/data_mart/projects.py

```diff
@@ -288,7 +274,16 @@ def _get_proprietary_proposed_offshore(engine: sa.engine.Engine) -> pd.DataFrame
         proj.developer,
         proj."capacity_mw",
         date(proj.proposed_completion_year::text || '-01-01') as date_proposed_online,
-        'active' as queue_status,
+        CASE coalesce(proj.construction_status, 'TBD') -- map to ISO queue_status categories
```

Comment:
I think we should just fill the missing `construction_status` values with TBD in the airtable sheet.

Reply:
We could do that, but then we have to fix new/changed values in Airtable with each update. To me `.fillna` seems easier to automate and never have to think about again

Reply:
Sounds good 👍